mkdocs.yml

```
@@ -99,8 +99,8 @@ extra:
   version:
     provider: mike
   var:
-    prev_git_tag: "v0.15.0"
-    chart_version: 0.10.1
+    prev_git_tag: "v0.15.2"
```

We are releasing v0.15.2 and the previous release is v0.15.1:
```suggestion
    prev_git_tag: "v0.15.1"
```